test/e2e/etcd_multi_node_test.go

```diff
@@ -138,6 +152,56 @@ var _ = Describe("Etcd", func() {
 	})
 })
 
+func deleteMemberDir(ctx context.Context, cl client.Client, logger logr.Logger, etcd *v1alpha1.Etcd, podName string) {
+	Expect(deleteDir(kubeconfigPath, namespace, podName, "backup-restore", "/var/etcd/data/new.etcd/member")).To(Succeed())
+	checkUnreadySts(ctx, cl, logger, etcd)
+	checkReadySts(ctx, cl, logger, etcd)
+}
+
+func deletePodAndCheckSts(ctx context.Context, cl client.Client, logger logr.Logger, etcd *v1alpha1.Etcd, podName string) {
+	pod := &corev1.Pod{}
+	ExpectWithOffset(1, cl.Get(ctx, types.NamespacedName{Name: podName, Namespace: "shoot"}, pod)).To(Succeed())
+	ExpectWithOffset(1, cl.Delete(ctx, pod, client.PropagationPolicy(metav1.DeletePropagationForeground))).To(Succeed())
+	checkUnreadySts(ctx, cl, logger, etcd)
+	checkReadySts(ctx, cl, logger, etcd)
+}
+
+func corruptMemberDBFile(ctx context.Context, cl client.Client, logger logr.Logger, etcd *v1alpha1.Etcd, podName string) {
+	Expect(corruptDBFile(kubeconfigPath, namespace, podName, "backup-restore", "/var/etcd/data/new.etcd/member/snap/db")).To(Succeed())
+	checkUnreadySts(ctx, cl, logger, etcd)
+	checkReadySts(ctx, cl, logger, etcd)
+}
+
+func checkUnreadySts(ctx context.Context, cl client.Client, logger logr.Logger, etcd *v1alpha1.Etcd) {
+	logger.Info("waiting for sts to become unready", "statefulSetName", "etcd-aws")
```

Comment:
Pls ignore `statefulSetName` info and pass [objLogger](https://github.com/gardener/etcd-druid/blob/master/test/e2e/etcd_multi_node_test.go#L69)